integration/todo/src/elements/todo-list.element.ts

```diff
@@ -9,11 +9,11 @@ import {
 } from '../services/todo.service.js';
 import { createTodoCard, TodoCardElement } from './todo-card.element.js';
 
-@injectable
-@element
+@injectable()
+@element({
+  tagName: 'todo-list'
```

Comment:
Is an element always injectable, or can be by default? 
Maybe that would simplify things... related to my comment in injetable.ts

Reply:
I have thought about that, would definitely be possible

Reply:
Thinking more about it I do t think I want elements to always be injectors. If someone wanted to do this they could pretty easily wrap both in their own decorator.